letsencrypt/config.json

```diff
@@ -9,19 +9,79 @@
   "boot": "manual",
   "ports": {
     "80/tcp": 80
-   },
+            },
   "map": ["ssl:rw"],
   "options": {
     "email": null,
     "domains": [null],
     "certfile": "fullchain.pem",
-    "keyfile": "privkey.pem"
+    "keyfile": "privkey.pem",
+    "challenge": null,
+    "dnsprovider": "",
```

Comment:
I think we should name it `dns_provider` because that will later rander as `dns provider` on UI